http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/api/TestLeoRoutes.scala

```diff
@@ -31,6 +37,14 @@ trait TestLeoRoutes { this: ScalatestRouteTest with Matchers with CommonTestData
   implicit private val nr = FakeNewRelicMetricsInterpreter
   implicit def unsafeLogger = Slf4jLogger.getLogger[IO]
 
+  val mockGoogleDirectoryDAO = new MockGoogleDirectoryDAO()
+  Await.result(
+    mockGoogleDirectoryDAO.createGroup(dataprocImageProjectGroupName,
+                                       dataprocImageProjectGroupEmail,
+                                       Option(mockGoogleDirectoryDAO.lockedDownGroupSettings)),
+    Duration.Inf
+  )
```

Comment:
This is to create the Google group ahead of testing. 

Does it make sense to factor it out a bit to a `def` like

```scala
def runAndWaitFor[T](f: Awaitable[T]): T = Await.result(f, Duration.Inf)
```
so it can be called as 
```scala
runAndWaitFor(mockGoogleDirectoryDAO.createGroup(
                   dataprocImageProjectGroupName,
                   dataprocImageProjectGroupEmail,
                   Option(mockGoogleDirectoryDAO.lockedDownGroupSettings)))
```
and put the `def` in a new `TestSupport` trait or something? Sam does this.

Reply:
Ah see above comment -- use ScalaTest `.futureValue`